configs/_base_/datasets/kittistep_368x368.py

```diff
@@ -0,0 +1,35 @@
+_base_ = './kittistep.py'
```

Comment:
Sorry, I didn't see any difference between the `kittistep.py` and `kittistep_368x368.py`
Would you like to tell us the difference?

Reply:
Yes you are right... There is no difference. ...368x368.py could be deleted!